install_config/install/docker_registry.adoc

```diff
@@ -141,8 +141,61 @@ $ oc volume deploymentconfigs/docker-registry \
 
 [NOTE]
 ====
-See link:#registry-known-issues[Known Issues] if using a scaled registry with a
-shared NFS volume.
+See link:#known-issue-nfs-image-push-fails[a known issue] if using a scaled
+registry with a shared NFS volume.
+====
+
+[[registry-amazon-s3-storage-back-end]]
+===== Use Amazon S3 as a storage back-end
+
+There's also an option to use Amazon Simple Storage Service storage with
+internal docker registry. It's a secure cloud storage manageable trough
```

Comment:
s/internal docker registry/the internal Docker registry
s/It's/It is
s/trough/through